### PR TITLE
/////////////''''''''''\ggggggggggggggggggggggggggggfffffffffffffffff…

### PR DESCRIPTION
**Pull request by noahdossan**
…fffffffffffffffd

g';flllllllllllllllllllllllllllllld'g;dffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff'[;hj

\h'jgggggggggghglf

\\\'\d'f Sticky keys have been disabled

### CODE REVIEW

**Comment by noahdossan** [comment]
Found my keyboard on the floor, sorry.

**Review by leofloripa — COMMENTED** [review]
@kadamski

**Comment by leofloripa** [comment]
@sugi

**Comment by michael-eddy** [comment]
rexbr7t8h978yt68879ty87hyyinmyf cxdruik,l;.ggfd rcazwsedrftfvgbyhnjkm,  rtfyujinkm 4drtfgvbhjn m

**Comment by vk8a8** [comment]
peak

**Review by leofloripa — CHANGES_REQUESTED** [review]
unnecessary change

File: README [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Linux kernel
+llllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllLinux kernel
```
Comment (leofloripa) [inline README:1]: Kerneeellllllllllllllllllll

**Pull request closed without merging**